src/components/Pollquote/index.mdx

```diff
@@ -0,0 +1,28 @@
+---
+name: Pull Quote
+route: /pull-quote
+---
+
+import { Playground, PropsTable } from 'docz'
+import PullQuote from '.'
+import { css } from 'react-emotion'
+
+# Pull Quote
+
+<PropsTable of={PullQuote} />
+
+<Playground>
+  <PullQuote
+    style={css`
+      svg {
+        margin-top: 10px;
```

Comment:
why do we have styles here?

Reply:
to demonstrate usage for the custom style prop